dra-evolution/testdata/pools-two-nodes-dgxa100.yaml

```diff
@@ -4,12658 +4,1118 @@
     creationTimestamp: null
     name: nvidia-00-dgxa100
   spec:
-    devices:
-    - attributes:
-      - name: uuid
-        string: GPU-684cbd68-c82a-4100-9fde-e63109a9fac3
-      - name: product-name
-        string: Mock NVIDIA A100-SXM4-40GB
-      - name: brand
-        string: Nvidia
-      - name: architecture
-        string: Ampere
-      - name: cuda-compute-capability
-        version: 8.0.0
-      - name: driver-version
-        version: 550.54.15
-      - name: cuda-driver-version
-        version: 12.4.0
-      - bool: true
-        name: mig-capable
-      - bool: false
-        name: mig-enabled
-      name: gpu-0
-      sharedCapacityConsumed:
+    deviceShape:
```

Comment:
This imposes a subtle new requirement, right?  All devices in a slice MUST be the same shape.

Reply:
Yes it does. I made a similar comment somewhere as well.

Reply:
Yes. Of course we could allow multiple shapes and reference then by name in the device list, but I don't see any real value in doing that over just having another slice.